pypit/artrace.py

```diff
@@ -1985,14 +2000,16 @@ def trace_tilt(slf, msarc, prefix="", tltprefix="", trcprefix=""):
         centval=-999999.9*np.ones((arcdet.shape[0],maskorder.size))
         nm=0
         for i in range(maskorder.size):
-            if maskorder[i] == 1: continue
-            w = arcdet[np.where(arcdet[:,nm]!=-1)[0],nm]
-            if np.size(w) != 0: centval[:np.size(w),i] = w
+            if maskorder[i] == 1:
+                continue
+            w = arcdet[np.where(arcdet[:, nm] != -1)[0], nm]
+            if np.size(w) != 0:
+                centval[:np.size(w), i] = w
             nm += 1
     elif settings.argflag['trace']['slits']['tilts']['method'] == 'fit1D':
         # Go along each order and fit the tilts in 1D
-        tiltang=-999999.9*np.ones((arcdet.shape[0],maskorder.size))
-        centval=-999999.9*np.ones((arcdet.shape[0],maskorder.size))
+        tiltang = -999999.9*np.ones((arcdet.shape[0], maskorder.size))
```

Comment:
Originally, I didn't think that cython dealt with np.nan, but I've just found out that it does (using the math.h include file). But, I'll save this for another PR, because there are many (unrelated) changes to the cython code.